web/ui/package.json

```diff
@@ -28,5 +28,5 @@
     "ts-jest": "^29.2.2",
     "typescript": "^4.9.5"
   },
-  "version": "0.54.1"
+  "version": "3.0.0-beta.0"
```

Comment:
Not sure about this change here. IF we follow the same version pattern as for go modules here this should be `0.300.0-beta.0` I believe. Same for the other locations.

Reply:
Changed to `0.300.0-beta.0` in https://github.com/prometheus/prometheus/pull/14853/commits/962ebee4f9b324ede8d3690e7e834a420e616e15

Using the new `get_module_version.sh` script to update the version when running `make ui-bump-version`

Reply:
Yeah, I think that should be fine for the UI as well. @Nexucis can confirm or deny, but I don't see anything different we should do there.

Reply:
yes that sounds right to me.